src/packages/math/src/euler.js

```diff
@@ -1,8 +1,8 @@
 import MathArray from './math-array';
 import {checkNumber, clamp} from './common';
-import {Matrix4} from './matrix4';
-import {Quaternion} from './quaternion';
-import {Vector3} from './vector3';
+import Matrix4 from './matrix4';
+import Quaternion from './quaternion';
+import Vector3 from './vector3';
```

Comment:
+1 for fixing this class, has been broken for a while.
But should be on a separate PR?